apps/examples/nextjs/auth.ts

```diff
@@ -53,6 +54,11 @@ const config = {
   adapter: UnstorageAdapter(storage),
   providers: [
     Apple,
+    Atlassian({
+      clientId: process.env.AUTH_ATLASSIAN_ID,
+      clientSecret: process.env.AUTH_ATLASSIAN_SECRET,
+      redirectUri: process.env.AUTH_ATLASSIAN_REDIRECT_URI,
+    }),
```

Comment:
```suggestion
    Atlassian,
```

We detect these automatically. Furthermore, redirectUri is not a valid value for a provider config.